src/Sulu/Bundle/AdminBundle/Admin/Admin.php

```diff
@@ -21,6 +21,7 @@
  */
 abstract class Admin implements ViewProviderInterface, NavigationProviderInterface
 {
+    const SULU_ADMIN_SECURITY_SYSTEM = 'Sulu';
```

Comment:
hope this is okay for you. would like to use a constant that describes the meaning of the value in the `Admin` classes. also,  the way i understood security systems, we should use a dedicated security system for admin related roles 🙂

Reply:
I would prefer not to introduce a new constant, because it is only introduced to avoid a bigger change... However, if you insist on that I am ok with it.

Reply:
i think using a constant makes things more understandable. otherwise, it is really hard to understand what the `'Sulu'` string means that is used in the `Admin` classes..

what bigger change do you mean? what disadvantages do you see?

Reply:
Explained it in the comment below. I am just not sure if this is the right place and if it should be a constant at all, so this contradicts a bit with a mentality of "Duplication is better than the wrong abstraction". But as I already repeatedly said, if you insist on it I can live with it. It's probably not the worst thing we are doing.

Reply:
sorry - i deleted my comment after i saw the answer below 🙈 as @alexander-schranz has approved this, i will dare to merge this and we can hopefully forget about this topic again 😅